service/src/main/resources/db/migration/V216__guardian_blocklist.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE guardian_blocklist (
+    guardian_id uuid NOT NULL CONSTRAINT fk$guardian REFERENCES person (id) ON DELETE CASCADE,
+    child_id uuid NOT NULL CONSTRAINT fk$child REFERENCES person (id) ON DELETE CASCADE
+);
+
```

Comment:
Is this feature intended to be used only via direct database access? Anyway, consider future proofing by adding the default `created` and `updated` columns and the trigger (maybe even evaka_user).